aten/src/TH/THAllocator.cpp

```diff
@@ -585,13 +577,13 @@ static void THRefcountedMapAllocator_free(void* ctx_, void* data) {
 void THRefcountedMapAllocator_incref(THMapAllocatorContext *ctx, void *data)
 {
   THMapInfo *map_info = (THMapInfo*)(((char*)data) - TH_ALLOC_ALIGNMENT);
-  THAtomicIncrementRef(&map_info->refcount);
+  ++map_info->refcount;
 }
 
 int THRefcountedMapAllocator_decref(THMapAllocatorContext *ctx, void *data)
 {
   THMapInfo *map_info = (THMapInfo*)(((char*)data) - TH_ALLOC_ALIGNMENT);
-  return THAtomicDecrementRef(&map_info->refcount);
+  return --map_info->refcount == 0;
```

Comment:
This change (and 3 other above) are no longer atomic. You haven't marked the `refcount` member as `std::atomic`, so it is unsafe now!